### PR TITLE
Fixed some bugs + added documentation + added a few features

### PR DESCRIPTION
**Pull request by JarnoWesthof**

### CODE REVIEW

File: WebHost.py [hunk 1]
```diff
@@ -48,6 +48,6 @@ def get_app():
     if app.config["SELFHOST"]:  # using WSGI, you just want to run get_app()
         if app.config["DEBUG"]:
             autohost(app.config)
-            app.run(debug=True, port=app.config["PORT"])
+            app.run(debug=True, port=90)
```
Comment (Ijwu) [inline WebHost.py]: This seems to be leftover from debugging, please confirm.

File: README.md [hunk 2]
```diff
@@ -10,6 +10,7 @@ Currently, the following games are supported:
 * Slay the Spire
 * Risk of Rain 2
 * The Legend of Zelda: Ocarina of Time
+* Timespinner
```
Comment (Ijwu) [inline README.md:13]: Yay!

File: WebHostLib/static/assets/gameInfo/en_Timespinner.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+## What items and locations get shuffled?
+All main inventory items, orbs, collectables, and familiers can be shuffled, and all locations in the game which could
+contain any of those items may have their contents changed.
+
+## Which items can be in another player's world?
+Any of the items which can be shuffled may also be placed into another player's world. It is possible to choose to
+limit certain items to your own world.
+
+## What does another world's item look like in Subnautica?
+Items belonging to other worlds are represented by the vanilla item [Elemental Beads](https://timespinnerwiki.com/Use_Items), Elemental Beads have no use in the randomizer
```
Comment (Ijwu) [inline WebHostLib/static/assets/gameInfo/en_Timespinner.md:24]: What an appropriate item for Archipelago, good choice!

Comment (Berserker66) [reply WebHostLib/static/assets/gameInfo/en_Timespinner.md:24]: `What does another world's item look like in Subnautica?` Subnautica?

Comment (Ijwu) [reply WebHostLib/static/assets/gameInfo/en_Timespinner.md:24]: I missed that, haha, woah. 🍝 Some pasta.

**Comment by Berserker66** [comment]
Thank you!

**Comment by Ijwu** [comment]
Feedback on game info page was addressed in commit 391f42b4f2357fb98de6125c8ad9c6dc4934d2a0

**Pull request merged**